lambdas.rst

```diff
@@ -10,7 +10,7 @@ normal functions and even behave like them.
 
 .. code:: python
 
-    lambda argument: manipulate(argument)
+    lambda arguments: expression
```

Comment:
I was trying to illustrate that you can pass multiple arguments. This is how the Python docs do it (although honestly I didn't confirm it until now). Happy to revert if you still don't agree. After all, it is your book! :)